parlai/scripts/interactive_rank.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+"""Does human evaluation on a task with label_candidates.
+
+Human can exit with ctrl + c and metrics will be computed and displayed.
+
+Examples
+--------
+
+.. code-block:: shell
+
+  python examples/interactive_rank.py -t babi:task10k:1 -dt valid
+
+When prompted, enter the index of the label_candidate you think is correct.
+Candidates are shuffled for each example.
+During datatype train, examples are randomly sampled with replacement; use
+train:ordered to not repeat examples.
+During datatype valid or test, examples are shown in order, not shuffled.
+"""
+from parlai.core.metrics import Metrics
+from parlai.core.params import ParlaiParser
+from parlai.core.agents import create_agent, create_task_agent_from_taskname
+
+import random
+
+
+def setup_args(parser=None):
+    if parser is None:
+        parser = ParlaiParser()
+    # parser.add_argument('-d', '--display-examples', type='bool', default=False)
```

Comment:
did you mean to get rid of these comments?

Reply:
ah yes I'll remove completely, thanks